ev-dev-tools/src/ev_cli/templates/interface-Exports.hpp.j2

```diff
@@ -3,13 +3,13 @@
 #ifndef {{ info.hpp_guard }}
 #define {{ info.hpp_guard }}
 
-{{ print_template_info('0.0.2') }}
+{{ print_template_info('1.0.0') }}
 
 #include <framework/ModuleAdapter.hpp>
 
 class {{ info.class_name }} {
 public:
-    {{ info.class_name }}(Everest::ModuleAdapter* ev, const std::string& name) : _ev(ev), _name(name){};
+    {{ info.class_name }}(Everest::ModuleAdapter* adapter, const std::string& requirement_id, const uint64_t idx) : _adapter(adapter), _requirement_id(requirement_id), _idx(idx){};
```

Comment:
Now that I see it, we probably want to change the interface to a custom requirement id type, like
struct RequirementID {
  std::string name;
  uint64_t index;
}

because it looks a bit strange this way, "one" requirement id should be enough

Reply:
Well,that would be a new language, because the string key in the config is called "requirement id" and this references it.
Having a new struct RequirementRef seems ok, though.